sql/core/src/main/scala/org/apache/spark/sql/catalyst/analysis/ResolveSessionCatalog.scala

```diff
@@ -595,6 +602,11 @@ class ResolveSessionCatalog(
       }
   }
 
+  private def getTableIdentifier(resolved: ResolvedTable): TableIdentifier = {
```

Comment:
This takes in `ResolveTable` instead of `Identifier` to be explicit that identifier is already resolved. (which is different from `asTableIdentifier` defined in `V2SessionCatalog.scala`. That's why I am also doing assert below.